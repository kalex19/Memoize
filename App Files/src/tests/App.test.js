// import React from 'react';
// import App from '../App/App.js';
// import { shallow } from 'enzyme';
// import { configure } from 'enzyme';
// import Adapter from 'enzyme-adapter-react-16';

// configure({ adapter: new Adapter() });

// describe('App', () => {
//     let wrapper;
  
//     beforeEach(() => {
//       wrapper = shallow(
//         <App
//         />
//       );
//     });

//     it('should match the snapshot with all data passed in', () => {
//         expect(wrapper).toMatchSnapshot();
//       });

//       it('should have the proper default state', () => {
//         expect(wrapper.state()).toEqual({ data: [] });
//       });

//       it('should have the proper default state', () => {
//         expect(wrapper.state()).toEqual({ questionSetKey: 1 });
//       });

//       expect(wrapper.state()).toEqual({ start: false });
//       expect(wrapper.state()).toEqual({ score: 0 });
//       expect(wrapper.state()).toEqual({ scoreAnimation: false }); 

//   });


