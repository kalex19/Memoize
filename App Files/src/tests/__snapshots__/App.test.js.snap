// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot with all data passed in 1`] = `
<div>
  <div
    className="App"
  >
    <header
      className="App-header"
    >
      <h1
        className="App-title"
      >
        CodeSnippits
      </h1>
    </header>
    <h3
      className="App-instructions"
    >
      Welcome! Analyze the code snippet. Select the correct answer. Submit to W!N.
    </h3>
    <h3
      className="App-hidden"
    >
      Select your answer. 
      <strong
        className="Text-bold"
      >
        Scroll
      </strong>
       to view options.
    </h3>
    <button
      className="App-btn"
      onClick={[Function]}
    >
      START
    </button>
  </div>
</div>
`;
